src/app/onboard/page.tsx

```diff
@@ -41,7 +41,16 @@ const Onboard: React.FC = () => {
         return;
       }
 
+      if (body.isDomainDuplicate) {
```

Comment:
We should be showing any error that is coming from the backend. I suggest we remove `isDomainDuplicate` because there are other parameters also that can cause an error. Just show the error coming from the backend.

See the error schema: `src/lib/schema/error.ts`